packages/devtools_app/lib/src/bloc/views/bloc_screen.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../../eval_on_dart_library.dart';
+import '../../globals.dart';
+import '../../screen.dart';
+import '../bloc/bloc_list_bloc.dart';
+import '../bloc/bloc_list_event.dart';
+import '../bloc/bloc_list_state.dart';
+import '../bloc/bloc_node_bloc.dart';
+import '../widgets/bloc_error_view.dart';
+import '../widgets/bloc_initial_view.dart';
+import '../widgets/bloc_list_view.dart';
+import '../widgets/bloc_loading_view.dart';
+
+// Entrypoint for the Bloc Screen in the DevTools UI
+class BlocScreen extends Screen {
+  const BlocScreen()
+      : super.conditional(
+            id: id,
+            requiresLibrary: 'package:flutter_bloc/',
+            title: 'Bloc',
+            icon: Icons.palette);
+
+  static const id = 'bloc';
+
+  @override
+  Widget build(BuildContext context) {
+    return const BlocScreenBody();
```

Comment:
can we move the contents of `BlocScreenBody.build` here?

Reply:
that would deviate from the approach provider uses